metadata/metadata-modeshape/src/main/java/com/thinkbiganalytics/metadata/modeshape/feed/JcrFeed.java

```diff
@@ -365,6 +365,10 @@ public void updateInitStatus(InitializationStatus status) {
         getFeedData().ifPresent(d -> d.updateInitStatus(status));
     }
 
+    public void removeInitStatus() {
+        getFeedData().ifPresent(d -> d.removeInitStatus());
+    }
+
     @Override
     public HistoryReindexingStatus getCurrentHistoryReindexingStatus() {
         return getFeedData().map(d -> d.getCurrentHistoryReindexingStatus()).orElse(null);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)